### DISCUSSION TITLE
What is Missing for Arm?

### DISCUSSION THREAD

**Discussion by monzeromer-hub** [rust-mobile/xbuild#195] [Q&A]
what is missing for adroideabi, arm-v7a, ...etc ??
I've noticed that the current arm processors for Android supports is limited to ARM64, based on this code:

```Rust
/// Returns the triple used by the non-LLVM parts of the NDK
    pub fn ndk_triple(self) -> &'static str {
        assert_eq!(self.platform(), Platform::Android);
        match self.arch() {
            Arch::Arm64 => "aarch64-linux-android",
            //Arch::Arm => "arm-linux-androideabi",
            //Arch::X86 => "i686-linux-android",
            Arch::X64 => "x86_64-linux-android",
        }
    }
    
    ```

**Comment by dvc94ch** [comment]
Think someone had a PR at some point. Shouldn't be hard to add

**Reply by MarijnS95** [reply]
@dvc94ch maybe you are thinking about duplicate issue #114, and "falsely-closing" pull request #115 which only implements `x86` support, not `arm` support which would together close out "32-bit suport".

**Reply by monzeromer-lab** [reply]
I saw #115 while I was looking into it, but didn't really notice #114 , mostly because I only looked at the PRs, not the issues.

I made a PR for android arm32, This is the first time I contribute to project so please let me know if I did something wrong to fix it asap, [arm32 support](https://github.com/rust-mobile/xbuild/pull/199) .
